test/test_average.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License
+# is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+import sockeye.average as average
+
+@pytest.mark.parametrize(
+    "test_points, expected_top_n, size, maximize", [
```

Comment:
thanks for adding these tests!
Could you modify the test cases so that they reflect better what a 'point' is, namely a tuple of [metric_value, checkpoint], i.e. having metric values that differ from checkpoint ids? Or in other words, explicitly test for conflicting ranking between metric value and checkpoint id to ensure that the strategy really looks at metric values (first element of the tuple)